packages/sakuli-legacy/src/context/sahi/action/mouse-action/mouse-actions-api.function.ts

```diff
@@ -26,19 +25,19 @@ export function mouseActionApi(
         throw Error('Not yet implemented due to api incompatibility');
     }
 
-    async function _click(query: SahiElementQueryOrWebElement, combo: string = ""): Promise<void> {
+    async function _click(query: SahiElementQueryOrWebElement, combo: string = "", options: {force:boolean}): Promise<void> {
```

Comment:
We should introduce a dedicated type of the options object instead of using a in-place object literal here. This would allow us to tweak the API in a way that it's possible to pass an options object either as second or third parameter, eliminating the superfluous empty string parameter to configure options.